scripts/sct_create_mask.py

```diff
@@ -110,12 +101,11 @@ def create_mask(param):
 
     path_tmp = sct.tmp_create(basename="create_mask", verbose=param.verbose)
 
-    sct.printv('\nCheck orientation...', param.verbose)
+    sct.printv('\nOrientation:', param.verbose)
```

Comment:
Hmmm, which PR do you want to merge first?

Reply:
i knew there would be a problem there... i started #2021, then found a bug which is solved by #2022, but then i needed the changes from #2022 to be able to continue working on #2021, so i did a rebase 2021-->2022 and here we are.